README.md

```diff
@@ -33,3 +33,10 @@ jekyll serve
 ## Contributing
 
 For guidelines on contributing to these documents, see [CONTRIBUTING.md](CONTRIBUTING.md).
+
+## Troubleshooting
+
+### jekyll not found
+If you encounter a problem while an installed **gems** that states that it can not be found, make sure that the directory in which **gems** are installed is included in your PATH.
```

Comment:
```suggestion
If you encounter a problem while running `bundle install` that states that Jekyll (or any other gem) cannot be found, make sure that the directory in which **gems** are installed is included in your PATH.
```

Reply:
This problem should actually be that the `~/.gems/bin` or other dirs were not added to the path. `bundle install` should run correctly when using the specified bundler version :) But I have updated the sentence. 
I hope it is more clear now?

Reply:
Cool - thanks, @b-m-f